BaragonUI/app/actions/api/services.es6

```diff
@@ -29,3 +31,20 @@ export const ReloadService = buildJsonApiAction(
     url: `/state/${serviceId}/reload`,
   })
 );
+
+export const RemoveUpstreams = buildJsonApiAction(
+  'REMOVE_UPSTREAMS',
+  'POST',
+  (loadBalancerService, upstreams, noValidate, noReload) => ({
+    url: '/request',
```

Comment:
there are several different actions that might submit a request. Maybe it makes sense to use the `SubmitRequest` action in `requests.e6` and let each of the submitting processes handle generating the necessary body?

Feel free to disagree, just trying to think how to best organize what might be very similar calls from different places.